tests/memset_sanity.c

```diff
@@ -17,6 +17,9 @@ int main(int argc, char *argv[]) {
     }
 
     p = (uint8_t *) iso_alloc(32);
+#if(__FreeBSD__ || __NetBSD__ || __OpenBSD__ || __DragonFly__)
+    bzero(p, 32);
```

Comment:
Why not create a new test just for `bzero` ?

Reply:
it was for simplicity sake :-)